### PR TITLE
Added example markdown for knit2synapse functionalities

### PR DESCRIPTION
**Pull request by th1vairam**
Looking at the recent inflow of new users, here are some sample markdown examples for them to use.

### CODE REVIEW

File: examples/example_createAndKnitToFileEntity.Rmd [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+library(knit2synapse) # Get the package from devtools::install_github('Sage-Bionetworks/knit2synapse')
+
+synLogin() # Login in to synapse before running knit2synapse
+
+knit2synapse::createAndKnitToFileEntity(file = "./example_createAndKnitToFolderEntity.Rmd", # This markdown file name
+                                        parentId = "syn117", # parentId to store this file in synapse
+                                        fileName = "Example for createFile") # optional fileName
+```
+
+## R Markdown
```
Comment (kdaily) [inline examples/example_createAndKnitToFileEntity.Rmd:39]: remove this boiler plate section

File: examples/example_createAndKnitToFileEntity.Rmd [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Create a synapse file entity, push the R markdown as file entity to synapse, and populate the synapse wiki with results"
```
Comment (kdaily) [inline examples/example_createAndKnitToFileEntity.Rmd:2]: shorter title

**Review by kdaily — CHANGES_REQUESTED** [review]
needs to go into a vignettes directory. See http://r-pkgs.had.co.nz/vignettes.html

**Comment by jaclynbeck-sage** [comment]
Merging with a new branch `vignette_updates` so I can address the comments before merging with master.

**Pull request merged**